src/components/extensive/WorkdayCollapseGrid/WorkdaySection.tsx

```diff
@@ -5,23 +5,32 @@ import { When } from "react-if";
 
 import Text from "@/components/elements/Text/Text";
 import WorkdayRow from "@/components/extensive/WorkdayCollapseGrid/WorkdayRow";
+import { Framework } from "@/context/framework.provider";
 
 import Icon, { IconNames } from "../Icon/Icon";
 import { useSectionData } from "./hooks";
-import { Demographic, DEMOGRAPHIC_TYPE_MAP, DemographicType, WorkdayGridVariantProps } from "./types";
-
+import {
+  Demographic,
+  DEMOGRAPHIC_TYPE_MAP,
+  DemographicType,
+  HBF_DEMOGRAPHIC_TYPE_MAP,
+  HBFDemographicType,
+  WorkdayGridVariantProps
+} from "./types";
 export interface WorkdaySectionProps {
   demographics: Demographic[];
-  type: DemographicType;
+  type: DemographicType | HBFDemographicType;
   variant: WorkdayGridVariantProps;
+  framework: Framework;
   onChange?: (demographics: Demographic[]) => void;
 }
 
-const WorkdaySection = ({ demographics, type, variant, onChange }: WorkdaySectionProps) => {
+const WorkdaySection = ({ demographics, type, variant, onChange, framework }: WorkdaySectionProps) => {
```

Comment:
Likewise, please don't pass new props in for things that are supplied by context. This component should use `useFrameworkContext` internally instead.